blog-posts/guardian-bundle-size/guardian-bundle-size.md

```diff
@@ -0,0 +1,64 @@
+---
+published: false
+title: 'Best practice (1): control the size of your Lambda function bundle'
+cover_image:
+description: ''
+tags: serverless, lambda, quality, AWS
+series:
+canonical_url:
+---
+
+_This article is part of a series on [Guardian][guardian], an open-source, highly configurable, automated best-practice audit tool for Serverless architectures._
+
+# The lighter, the better
+
+A light bundle will positively impact your Serverless architecture for at least two reasons.
+
+## 1. A light deployment package reduces the cold start of your Lambda function
+
+The Serverless Framework automatically bundles your code and uploads it to AWS. The underlying tools are [serverless-webpack][serverless-webpack] or [serverless-esbuild][serverless-esbuild], which should be preferred for its shorter configuration and better performance.
+
+Not only does the size impact the time it takes to upload the bundle, it also significantly impacts the cold start.
+
+![Cold start durations](./assets/bundle-size-impact-on-cold-start.png 'Cold start durations per deployment size (https://mikhail.io/serverless/coldstarts/aws/#does-package-size-matter)')
+
+## 2. AWS enforces quotas on deployment resources
+
+Smaller bundles will help you respect [AWS quotas][quotas].
```

Comment:
On peut rajouter une phrase pour expliquer vite fait: `Smaller bundles equals more lambdas in your AWS account!`